woocommerce/woocommerce-gateway-stripe
# Remove duplicate banner

Fixes #3561 

When the legacy checkout is enabled, 2 different banners are displayed with a similar message.

## Changes proposed in this Pull Request:
Showing 'LegacyExperienceTransitionNotice' only when the promotional banner is not about the same.

## Testing instructions
- When the promotional surface is about re-auth, the yellow banner about enabling the new checkout experience should be displayed.

<img width="781" alt="Screenshot 2024-10-30 at 6 01 05 PM" src="https://github.com/user-attachments/assets/0e1f81e2-c991-4385-831e-236d1cb8c8b8">

- When the promotional surface is about the new checkout experience, the yellow banner about enabling the new checkout experience should not be there.

<img width="781" alt="Screenshot 2024-10-30 at 6 00 39 PM" src="https://github.com/user-attachments/assets/77556853-55f6-4963-b4d3-2a27e8d7ba53">
